src/storage/QueryBaseProcessor.inl

```diff
@@ -207,8 +210,7 @@ bool QueryBaseProcessor<REQ, RESP>::checkExp(const Expression* exp) {
         case Expression::kEdgeType: {
             return true;
         }
-        case Expression::kAliasProp:
-        case Expression::kEdgeProp: {
```

Comment:
Why remove kEdgeProp?

Reply:
There is already no EdgePropertyExpression yet.  

FYI. [PR669](https://github.com/vesoft-inc/nebula/pull/669)

Reply:
Good